pages/deployment/[uuid].tsx

```diff
@@ -162,14 +166,40 @@ export default function Deployment() {
     await postEndpoint(`/api/v1/deployment/${deployment?.uuid}/reset-approvals`, {}).then((res) => res.json())
   }
 
+  const initialVersionRequested: ObjectId | VersionDoc = deployment.versions.filter(
+    (version: any) => version.version === deployment.metadata.highLevelDetails.initialVersionRequested
+  )[0] as VersionDoc
+
   return (
     <>
       <Wrapper title={`Deployment: ${deployment.metadata.highLevelDetails.name}`} page='deployment'>
-        <Box sx={{ textAlign: 'right', pb: 3 }}>
-          <Button variant='outlined' color='primary' startIcon={<Info />} onClick={handleClickOpen}>
-            Show download commands
-          </Button>
-        </Box>
+        {initialVersionRequested.metadata.buildOptions.uploadType === undefined ||
```

Comment:
Instead of doing this long check multiple times, you could use the following:
```
const hasUploadType = useMemo(() => !!initialVersionRequested.metadata.buildOptions.uploadType, [initialVersionRequested])
```
Then in place of `initialVersionRequested.metadata.buildOptions.uploadType === undefined` you can do `!hasUploadType`